### ISSUE TITLE
[Bug]: Combobox display list options keyboard accessibility issue (VoiceOver)

### ISSUE THREAD

**Issue by helmines25** [radix-vue/radix-vue#1978]
### Environment

```bash
Development/Production OS: macOs Sequoia Version 15.5
Browser: Chrome Version 137.0.7151.69, Safari Version 18.5 (20621.2.5.11.8)
```

### Link to minimal reproduction

https://reka-ui.com/docs/components/combobox

### Steps to reproduce

1. Open VoiceOver and navigate to the example Combobox-element. 
2. Open a list of options via keyboard (Control-Option-Space).
3. Nothing happens - keyboard command should open the dropdown menu. 

### Describe the bug

VoiceOver instructs the user to either type text or to press Control-Option-Space to display a list of options. When the key command is used however, the list of choices is not displayed visually nor does VoiceOver announce the content of the list to the user. As a result, screen reader users are unable to access or navigate the list.

Tested on Chrome Version 137.0.7151.69 and Safari Version 18.5 (20621.2.5.11.8)

### Expected behavior

Voiceover's keyboard command should open the dropdown menu.

### Context & Screenshots (if applicable)

<img width="1240" alt="Image" src="https://github.com/user-attachments/assets/b8d3f3f5-da32-4cec-a711-6b9a1ae71961" />
